build.gradle

```diff
@@ -455,9 +455,14 @@ task bundle(type: Zip) {
     from("gatk")
     from("README.md")
     from("build/docs/tabCompletion/gatk-completion.sh")
-    from("scripts/gatkcondaenv.yml")
-    from("$buildDir/gatkPythonPackageArchive.zip")
+    from("${buildDir}/${pythonPackageArchiveName}")
+    into(baseName)
```

Comment:
Is this second `into(baseName)` necessary?  I'm not totally sure how the from/ into interacts I guess, but it looks like this might be an unintentional addition?

Reply:
It was intentional, but you're right that its not needed. I had previously been doing

`from("${buildDir}/${pythonPackageArchiveName}").filter(...)`

which corrupted the entire zip file. Using the `from` overload that takes the closure fixes that, but now I can remove the superfluous `into`.

Reply:
I find this stuff very confusing in gradle...  I wish they had a big list of examples of using some of these things.